### DISCUSSION TITLE
Window Uniformity

### DISCUSSION THREAD

**Discussion by exten** [helix-editor/helix#8210] [General]
Hi，First of all, I like this editor very much.But I think window uniformity is good for reading information, for example：
Is it possible to summarize shortcut windows and other information windows into the command line window to achieve window unification

**Comment by pascalkuthe** [comment]
i never head of window unification before so you are going to have to elaborate. Helix doesn't really have concept of windows so your examples are not really clear. What is the command line window?

**Reply by exten** [reply]
Thanks for your reply！
ex：
1. When you enter `:`   ---  The window pops up at the bottom
2. When you enter a `space`   ---  The window pops up in the lower right corner
3. When you enter a `g`   ---  The window pops up in the lower right corner
4. When running `run-shell-command`  --- The window is floating pop-up and does not fix a position

Whether this information can be displayed in a pop-up window as if a colon was entered. A uniform position, the same width, and height change depending on the amount of content. ：）
